src/components/map/MapService.js

```diff
@@ -1492,10 +1492,15 @@ goog.require('ga_urlutils_service');
         },
 
         flyToAnimation: function(ol3d, center, destination, defer) {
+          // In degrees
+          var pitch = 50;
+          // Default camera field of view
+          var cameraFieldOfView = 60;
```

Comment:
https://cesiumjs.org/Cesium/Build/Documentation/Camera.html
In the first example, I ll add it to the code.